js/components/Font.js

```diff
@@ -0,0 +1,11 @@
+const React = require("react");
+
+class Font extends React.Component {
+	render() {
+		return (
+			<Font style = {{color: 'blue'}}/>
```

Comment:
Neat! You looked up style stuff! That was something I wanted to avoid for a bit though, the `<font>` tag supports a `color="some color name"` property.